pkg/proxy/register.go

```diff
@@ -73,6 +74,9 @@ func (p *Register) Add(route *Route) error {
 }
 
 func (p *Register) createDirector(proxyDefinition *Definition, balancer Balancer) func(req *http.Request) {
+	parameterNameExtractor := router.NewListenPathParameterNameExtractor()
```

Comment:
I think we could reduce the name a bit, smth like `paramNameExtractor 
 := NewListenPathParamNameExtractor`